packages/integration-tests/tests/ui/transactions.spec.ts

```diff
@@ -1,17 +1,19 @@
 import { expect, test } from "@playwright/test";
 
 import { config } from "./config";
-import { BlockExplorer, Buffer } from "../../src/entities";
+import { BlockExplorer, Buffer, Token, Wallets } from "../../src/entities";
 import { Helper } from "../../src/helper";
 
 import type { Locator } from "@playwright/test";
 
 const bufferRoute = "src/playbook/";
 const helper = new Helper();
+const txSum = "0.000009";
```

Comment:
Is this a common value, should we use entities.ts for this?

Reply:
export enum CustomValue {
  txSum = "0.000009",
}

--> entities.ts

Reply:
![image](https://github.com/matter-labs/block-explorer/assets/51121511/06064fa1-4db3-4e7b-814f-3f8d850f41e6)